osf_tests/test_archiver.py

```diff
@@ -478,7 +479,7 @@ def test_archive_node_pass(self, mock_archive_addon):
             'osfstorage',
             self.archive_job._id
         )
-        assert (mock_group.called_with(archive_osfstorage_signature))
+        mock_group.assert_called_with(archive_osfstorage_signature)
```

Comment:
I think I might be wrong during our discussion, I believe the `mock_file_tree` and `mock_group` are accessible outside the `with` context. See this [link](https://stackoverflow.com/questions/6432355/variable-defined-with-with-statement-available-outside-of-with-block/6432414#6432414). So, it seems to be a correct test before the upgrade.

Please take a look at all similar/same cases that are skipped.

Reply:
The root of the issue is not related to context manager. The only thing which changes after an exit from context manager is that patch is undone. Mock object retains all the information about calls executed on it after closure of patch's context manager.

Reply:
ah, make sense